gdsfactory/components/cmos_side_gate_transistor.py

```diff
@@ -0,0 +1,238 @@
+"""
+CMOS side-gate transistors from
+F. Zanetto et al., “Unconventional Monolithic Electronics in a
+Conventional Silicon Photonics Platform,” IEEE Trans. Electron Devices,
+vol. 70, no. 10, pp. 4993-4998, Oct. 2023, doi: 10.1109/TED.2023.3304268.
+"""
+
+from __future__ import annotations
+
+import gdsfactory as gf
+from gdsfactory.cell import cell
+from gdsfactory.component import Component
+
+# TODO: find correct imports
+
+
+@cell
+def cmos_side_gate(
+    is_nmos: bool = True,
```

Comment:
I would expose here all the Pcell parameters here so that can be modified and reused for different processes

`t_ox`, `M1` ...